pkg/sr/serde.go

```diff
@@ -134,6 +131,26 @@ func (s *Serde) SetDefaults(opts ...SerdeOpt) {
 	s.defaults = opts
 }
 
+// DecodeID decodes an ID from in, returning the ID and the remaining bytes,
```

Comment:
`decodes an ID from b` but I can sneak this into a quickfix PR that I plan to do